backend/internal/router/middleware.go

```diff
@@ -3,7 +3,7 @@ package router
 import (
 	"net/http"
 	"strings"
-	. "threadule/backend/internal/presentation"
+	"threadule/backend/internal/presentation"
```

Comment:
that was actually on purpose so the the Response-functions look the same as in the presentation package

Reply:
oh.. i See. I revert it.